contentcuration/contentcuration/management/commands/restore_channel.py

```diff
@@ -0,0 +1,216 @@
+import sqlite3
+import sys
+import os
+import datetime
+from django.conf import settings
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.db import transaction
+from le_utils.constants import content_kinds,file_formats, format_presets, licenses, exercises
+from contentcuration import models
+from contentcuration.api import write_file_to_storage
+import logging as logmodule
+logging = logmodule.getLogger(__name__)
+
+CHANNEL_TABLE = 'content_channelmetadata'
+NODE_TABLE = 'content_contentnode'
+FILE_TABLE = 'content_file'
+TAG_TABLE = 'content_contenttag'
+NODE_TAG_TABLE = 'content_contentnode_tags'
+LICENSE_TABLE = 'content_license'
+NODE_COUNT = 0
+FILE_COUNT = 0
+TAG_COUNT = 0
+
+
+class EarlyExit(BaseException):
+    def __init__(self, message, db_path):
+        self.message = message
+        self.db_path = db_path
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('channel_id', type=str)
```

Comment:
I think there's also a way to add a description for what the management command does. Can you look that up and add that as well?